gdx/src/com/badlogic/gdx/input/RemoteSender.java

```diff
@@ -173,6 +173,11 @@ public boolean touchUp (int x, int y, int pointer, int button) {
 		return false;
 	}
 
+	@Override
+	public boolean touchCancelled (int screenX, int screenY, int pointer, int button) {
+		return touchUp(screenX, screenY, pointer, button);
```

Comment:
This class is rarely used, but I am not sure if this behaviour is the correct one here

Reply:
Haha, you've spotted it... yeah, me neither. My thought here was, before a `touchUp` was always expected after a `touchDown` so, to keep backwards compatibility considering I have no idea what's the use cas of this class I pass it on `touchCancel` not to break anything.

Reply:
It is for controlling a game from a remote device. A weird feature that probably could be removed.